src/ray/raylet/node_manager.cc

```diff
@@ -1165,11 +1165,11 @@ void NodeManager::HandleNotifyGCSRestart(rpc::NotifyGCSRestartRequest request,
   // registered to raylet first (blocking call) and then connect to GCS, so there is no
   // race condition here.
   gcs_client_->AsyncResubscribe();
-  auto workers = worker_pool_.GetAllRegisteredWorkers(true);
+  auto workers = worker_pool_.GetAllRegisteredWorkers(/* filter_dead_worker */ true);
```

Comment:
Is this relevant to this PR?

Reply:
no, just want to make things consistent along the way while I was going over the call sites